src/main/java/jarvis/commands/Task.java

```diff
@@ -0,0 +1,55 @@
+package jarvis.commands;
+
+import jarvis.display.DisplayMessages;
+
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    public String getStatusIcon() {
+        // mark done task with X
+        return (isDone ? "X" : " ");
+    }
+
+    public String getDescription() {
+        return description;
+    }
+
+    public String getTypeIcon() { return "T"; }
```

Comment:
**Coding standard violation:** Separate the function definition into multiple lines